test/containers/HomePageSpec.js

```diff
@@ -1,34 +1,26 @@
 import test from 'tape'
 import React from 'react'
 import { Provider } from 'react-redux'
-import { createStore } from 'redux'
 import { MemoryRouter } from 'react-router-dom'
-
-import rootReducer from '../../src/reducers'
+import { mount } from 'enzyme'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'enzyme'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->